lib/kubernetes-deploy/kubernetes_resource/deployment.rb

```diff
@@ -168,16 +167,27 @@ def find_latest_rs(deployment_data)
     end
 
     def min_available_replicas
```

Comment:
I need to rework this function I dropped in the value from the annotation, but that’s not the right.

Reply:
Fixed the logic. Ready for 👀